GitVersionCore.Tests/Mocks/MockCommit.cs

```diff
@@ -6,12 +6,17 @@
 [DebuggerDisplay("{DebuggerDisplay}")]
 public class MockCommit : Commit
 {
+    static int commitCount = 1;
+    static DateTimeOffset when = DateTimeOffset.Now;
+
     public MockCommit(ObjectId id = null)
     {
         idEx = id ?? new ObjectId(Guid.NewGuid().ToString().Replace("-", "") + "00000000");
-        MessageEx = "";
+        MessageEx = "Commit " + commitCount++;
         ParentsEx = new List<Commit> { null };
-        CommitterEx = new Signature("Joe", "Joe@bloggs.net", DateTimeOffset.Now);
+        CommitterEx = new Signature("Joe", "Joe@bloggs.net", when);
+        // Make sure each commit is a different time
+        when = when.AddSeconds(1);
```

Comment:
Not really, if two tests get the same time that is not an issue. It is within the same test we want it to bump